src/fragrance_rater/services/recommendation_service.py

```
@@ -0,0 +1,342 @@
+                "accords": accord_score,
+                "family": family_score,
+                "subfamily": subfamily_score,
+                "raw": raw_score,
+            },
+        )
+
+    async def get_recommendations(
+        self,
+        reviewer_id: str,
+        limit: int = 10,
+        exclude_rated: bool = True,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)